src/py42/clients/__init__.py

```diff
@@ -1,6 +1,6 @@
-class BaseClient(object):
+from collections import namedtuple
 
-    __slots__ = ["_session"]
-
-    def __init__(self, session):
-        self._session = session
+Clients = namedtuple(
```

Comment:
These are in the wrong order from how it gets constructed causing nothing to work for me

Reply:
Perhaps this the fault of one of the formatters?

Reply:
the order of these shouldn't matter, are you observing otherwise?